src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -54,6 +57,7 @@ public EmployeeDto getEmployeeByUserName(String username) {
   }
 
   @Override
+  @Transactional(isolation = Isolation.SERIALIZABLE, propagation = Propagation.REQUIRES_NEW)
   public EmployeeDto updateEmployeeEntity(EmployeeDto employeeDto) {
```

Comment:
I think you need to change the propagation level

Reply:
Done